accelerator/mlu_accelerator.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Don't you need to add `mlu` references 
1. [here](https://github.com/microsoft/DeepSpeed/blob/9b7fc5452471392b0f58844219fcfdd14a9cdc77/accelerator/real_accelerator.py#L23) 
2. [here](https://github.com/microsoft/DeepSpeed/blob/9b7fc5452471392b0f58844219fcfdd14a9cdc77/accelerator/real_accelerator.py#L51)

Reply:
added